### ISSUE TITLE
Points breakdown data seems incorrect

### ISSUE THREAD

**Issue by shawnlauzon** [Volentix/venue-server#127]
I'm looking at data in the test database, and the numbers aren't adding up. I'm using screenshots to display, but I believe the problem is in the server.

The # of posts with the bonus percentage I get from the `points-breakdown`. But if I manually do the multiplication and add up the values, I get 2,232, not 3,131 as is displayed
![image](https://user-images.githubusercontent.com/611529/42919574-9503a3d2-8ae0-11e8-86fb-a388afd57710.png)
although I'm getting from the server:
```
  "sitewide_stats": {
    "total_posts": 31,
    "total_post_points": 3100,
    "total_bonus_points": 31,
```
which is actually also odd, because I see that the number of total posts is:
![image](https://user-images.githubusercontent.com/611529/42919762-811b3d70-8ae1-11e8-90d7-33c7c4f42256.png) 
and 22 is the correct data.

And this as well is wrong; 100*14.37 is not 2,020
![image](https://user-images.githubusercontent.com/611529/42919704-2d9ccc22-8ae1-11e8-8887-4301863f2c9f.png)
However I think that math might be wrong because 14.37 is wrong because the 3131 is wrong.

In short, I suspect there's just a few pieces of data which are wrong that is throwing off the whole thing, but I'm not sure which those pieces are exactly. Hope @joemarct can find out :)
